clients/js/src/ChromaFetch.ts

```diff
@@ -59,7 +59,7 @@ export const chromaFetch: FetchAPI = async (
       switch (resp.status) {
         case 400:
           throw new ChromaClientError(
-            `Bad request to ${input} with status: ${resp.statusText}`,
+            `Bad request to ${input} with message: ${respBody?.message}`,
```

Comment:
Why this change?

Reply:
Test was previously not running - this is now correct.

Reply:
bc previous test wasn't triggered